core/controllers/base.py

```diff
@@ -189,9 +189,9 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
             if self.user_id is None else user_settings.role)
         self.actions = role_services.get_all_actions(self.role)
 
-        rights_mgr_user = rights_manager.Actor(self.user_id)
-        self.is_moderator = rights_mgr_user.is_moderator()
-        self.is_admin = rights_mgr_user.is_admin()
+        self.is_moderator = bool(
```

Comment:
Why are the checks for this and self.is_admin treated differently (one is in terms of role, the other in terms of actions)?